typed-values/src/test/java/org/camunda/bpm/engine/test/variables/FileValueTypeImplTest.java

```diff
@@ -107,6 +107,12 @@ public void createValueFromBytes() throws IOException, URISyntaxException {
     assertThat(value.getType(), is(instanceOf(FileValueTypeImpl.class)));
     checkStreamFromValue(value, "text");
   }
+  
+  @Test(expected = IllegalArgumentException.class)
+  public void createValueFromObject() throws IOException, URISyntaxException {
+    type.createValue(new Object(), Collections.<String, Object> singletonMap(FileValueTypeImpl.VALUE_INFO_FILE_NAME, "simpleFile.txt"));
+    fail();
```

Comment:
unnecessary code because the test will always fail if the expected exception is not thrown